diff -Naur blankon-installer-1.29.orig/src/installer.vala blankon-installer-1.29/src/installer.vala
--- blankon-installer-1.29.orig/src/installer.vala	2016-07-29 09:36:43.153199644 +0700
+++ blankon-installer-1.29/src/installer.vala	2016-07-29 09:42:40.672569738 +0700
@@ -548,7 +548,7 @@
             Log.instance().log ("\nmount separated home partition\n");
             DirUtils.create ("/target/home", 0700);
             string [] c = { "/bin/mount", home, "/target/home" };
-            do_simple_command_with_args (c, Step.MOUNTHOME, "mounting_home_filesystem ", "Unable to mount home filesystem");
+            
         
             // write fstab file at tmp, will be copied to /target/etc/fstab by b-i-setup-fs script
             var root_partition = backtick("/bin/lsblk -no UUID " + partition_path);
@@ -557,12 +557,16 @@
             var content = "UUID=" + root_partition + " / ext4 defaults 1 2\n";
                content += "UUID=" + home_partition + " /home ext4 defaults 1 2\n";
             Utils.write_simple_file ("/tmp/fstab", content);
+
+            do_simple_command_with_args (c, Step.MOUNTHOME, "mounting_home_filesystem ", "Unable to mount home filesystem");
         } else if (secureInstall == false) {
             // write fstab file at tmp, will be copied to /target/etc/fstab by b-i-setup-fs script
             var root_partition = backtick("/bin/lsblk -no UUID " + partition_path);
 
             var content = "UUID=" + root_partition + " / ext4 defaults 1 2\n";
             Utils.write_simple_file ("/tmp/fstab", content);
+            last_step = Step.MOUNTHOME;
+             do_next_job ();
         } else {
             last_step = Step.MOUNTHOME;
             do_next_job ();
