diff -Naur blankon-installer-1.38.orig/system/src.js blankon-installer-1.38/system/src.js
--- blankon-installer-1.38.orig/system/src.js	2016-12-29 13:49:45.724898311 +0700
+++ blankon-installer-1.38/system/src.js	2016-12-31 09:32:40.418206409 +0700
@@ -1570,7 +1570,7 @@
   low_memory_warning : "Your physical memory is under 1 GB, which doesn't meet minimum requirements. Installation can't be continued.",
   low_partition_size_warning : "Your selected partition is under 8 GB, which doesn't meet minimum requirements. Please select another bigger partition or installation can't be continued.",
   low_disk_size_warning : "Your selected disk is under 8 GB, which doesn't meet minimum requirements. Please select another bigger disk or installation can't be continued.",
-  partoedi_warning : "WARNING : Advanced partition feature is still in development. You may found bug(s) and lead to data loss. If so, please report to http://http://dev.blankon.id/newticket."
+  partoedi_warning : "WARNING : Advanced partition feature is still in development. You may found bug(s) and lead to data loss. If so, please report to http://dev.blankon.id/newticket."
 }
 
 var id = {
@@ -1656,5 +1656,5 @@
   low_memory_warning : "Memori komputer Anda berukuran di bawah 1 GB, tidak memenuhi standar minimal. Pemasangan tidak dapat dilanjutkan.",
   low_partition_size_warning : "Partisi yang Anda pilih berukuran di bawah 8 GB, tidak memenuhi standar minimal. Pilih partisi lain yang lebih besar atau pemasangan tidak dapat dilanjutkan.",
   low_disk_size_warning : "Diska yang Anda pilih berukuran di bawah 8 GB, tidak memenuhi standar minimal. Pilih diska lain yang lebih besar atau pemasangan tidak dapat dilanjutkan.",
-  partoedi_warning : "PERINGATAN : Fitur dalam moda mahir masih dalam pengembangan. Anda mungkin akan menemui beberapa kutu dan kesalahan yang dapat berujung pada kehilangan data. Jika itu terjadi, silakan laporkan  http://http://dev.blankon.id/newticket."
+  partoedi_warning : "PERINGATAN : Fitur dalam moda mahir masih dalam pengembangan. Anda mungkin akan menemui beberapa kutu dan kesalahan yang dapat berujung pada kehilangan data. Jika itu terjadi, silakan laporkan  http://dev.blankon.id/newticket."
 }
