--- blankon-installer-1.23.orig/scripts/b-i-setup-fs	2016-07-12 19:32:52.252876755 +0700
+++ blankon-installer-1.23/scripts/b-i-setup-fs	2016-07-12 19:41:56.478865818 +0700
@@ -90,6 +90,18 @@
 ff02::2 ip6-allrouters
 @@EOF
 
+###### SCIM
+
+cat << @@EOF > $ROOTFS/target/etc/X11/Xsession.d/99scim
+
+GTK_IM_MODULE=scim
+QT_IM_MODULE=scim
+
+export GTK_IM_MODULE
+export QT_IM_MODULE
+@@EOF
+
+
 ###### Locale
 if [ -f /run/locale ];then
     cp /run/locale $ROOTFS/target/etc/default/
