diff -Naur blankon-installer_1.28.orig/scripts/b-i-install-grub blankon-installer_1.28/scripts/b-i-install-grub
--- blankon-installer_1.28.orig/scripts/b-i-install-grub	2016-07-27 23:03:11.887015591 +0700
+++ blankon-installer_1.28/scripts/b-i-install-grub	2016-07-27 23:05:57.147516355 +0700
@@ -26,7 +26,7 @@
 then
     # The GRUB_DISABLE_SUBMENY=y fix broken grub.cfg generation
     cat << @@EOF >> /target/etc/default/grub
-GRUB_CMDLINE_LINUX_DEFAULT="rw quiet cryptopts=target=root,source=/dev/lvm/root,lvm=root CRYPTSETUP=y"
+GRUB_CMDLINE_LINUX_DEFAULT="rw quiet splash cryptopts=target=root,source=/dev/lvm/root,lvm=root CRYPTSETUP=y"
 GRUB_GFXPAYLOAD_LINUX=1024x768
 @@EOF
 
diff -Naur blankon-installer_1.28.orig/scripts/b-i-setup-fs blankon-installer_1.28/scripts/b-i-setup-fs
--- blankon-installer_1.28.orig/scripts/b-i-setup-fs	2016-07-27 23:03:11.887015591 +0700
+++ blankon-installer_1.28/scripts/b-i-setup-fs	2016-07-27 23:04:58.089905717 +0700
@@ -50,7 +50,7 @@
   touch $ROOTFS/target/etc/crypttab
   echo "root /dev/lvm/root none luks,retry=1" > $ROOTFS/target/etc/crypttab
   touch $ROOTFS/target/etc/fstab
-  echo "/dev/mapper/root ext4 relatime,errors=remount-ro 0 1" >> $ROOTFS/target/etc/fstab
+  echo "/dev/mapper/root / ext4 relatime,errors=remount-ro 0 1" >> $ROOTFS/target/etc/fstab
   echo "/dev/mapper/lvm-swap    none    swap    sw  0   0" >> $ROOTFS/target/etc/fstab
   echo "$BOOT /boot vfat rw,relatime,fmask=0022,dmask=0022,codepage=437,iocharset=iso8859-1,shortname=mixed,errors=remount-ro 0 2" >> $ROOTFS/target/etc/fstab
   
